setup.sh

```diff
@@ -40,7 +40,3 @@ else
     download_docker_compose_binary "$PLATFORM" "$ARCH" "$DOCKER_COMPOSE_VERSION"
 fi
 
-source ./dev-scripts/utils.sh
-source ./dev-scripts/compile.sh
-compile_command
- 
```

Comment:
This was a hack to get the CI to build all the binaries (we have two now)

Reply:
Just make sure it's not breaking other dev local use cases, cc @chiptus as I know he is using this script.

Reply:
It was my hack.  I added it two nights ago so should be ok